cmd/vault/sign_settlement.go

```diff
@@ -317,9 +337,13 @@ func createUpholdArtifact(
 
 	// Ensure that there is only one payment for a given Uphold wallet by combining
 	// multiples if they exist.
-	flattenedTransactions := upholdsettlement.FlattenPaymentsByWalletProviderID(
-		&upholdOnlySettlements,
-	)
+	mergeCustodial, ok := ctx.Value("mergeCustodial").(bool)
+	fmt.Printf("%t", mergeCustodial)
```

Comment:
we can yank this once testing is complete, right?

Reply:
yes